script/testing/junit/traces/insert.test

```diff
@@ -231,3 +231,31 @@ SELECT * from insert12
 statement ok
 DROP TABLE insert12
 
+# Test inserts that are: out of order, missing values, multiple values, multiple types. Issue #1408
+statement ok
+CREATE TABLE insert13 (c1 integer, c2 double)
+
+statement ok
+INSERT INTO insert13 (c2, c1) VALUES (2.0, 1)
+
+query IR nosort
+SELECT * FROM insert13
+----
+1
+2.0
+
+statement ok
+INSERT INTO insert13 (c2) VALUES (4.0)
```

Comment:
Actually can you also add a multi-tuple insert like in #1408?

Reply:
oops, felt like I forgot something. Thanks for the catch!